test/types/enum/operations/order/enumOrder4.chpl

```diff
@@ -0,0 +1,20 @@
+enum color { red, green, blue};
+
+proc test(c: color) {
+  const order = chpl__enumToOrder(c);
+  writeln(c, " = ", order, " = ", chpl__orderToEnum(order, color));
+}
+
+proc test(param c: color, param val: int) {
+  if (chpl__enumToOrder(c) != val) then
+    compilerError("Shouldn't get here");
+  writeln("Param test passed");
+  test(c);
+}
+
+test(color.red, 0);
+test(color.green, 1);
+test(color.blue, 2);
+
+// try an illegal case:
+writeln(chpl__orderToEnum(4, color));
```

Comment:
Ditto for the param version as well.

Reply:
Commit https://github.com/chapel-lang/chapel/pull/17504/commits/ed214bc096bb958a19605a6c0269be1ce3e4508a does this using a config param rather than `4`, setting it to `-1` and `4`.